com.unity.template.renderstreaming-hd/Assets/Scripts/RenderStreaming.cs

```diff
@@ -57,6 +74,67 @@ public class RenderStreaming : MonoBehaviour
         private MediaStream videoStream;
         private MediaStream audioStream;
 
+        // Helper function for getting the command line arguments
+        private static string GetArg(string name)
+        {
+            var args = System.Environment.GetCommandLineArgs();
+            for (int i = 0; i < args.Length; i++)
+            {
+                if (args[i] == name && args.Length > i + 1)
+                {
+                    return args[i + 1];
+                }
+            }
+            return null;
+        }
+
+        IEnumerator GetRTCIceServers(System.Action<RTCIceServer[]> result)
+        {
+            List<RTCIceServer> serverList = new List<RTCIceServer>();
+
+            // Add servers from config
+            foreach (var server in iceServers)
```

Comment:
Can we simplify this to:
`List<RTCIceServer> serverList = new List<RTCIceServer>(iceServers);`
Or something similar so that we don't have to loop ?
 (Untested)